docs/src/index.md

```diff
@@ -288,62 +284,70 @@ This was once how all neural network code worked.
 
 Using ChainRules directly also helps get a feel for it.
 
-```jldoctest
-using ChainRules
+```jldoctest index; output=false
+using ChainRulesCore
 
 function foo(x)
     a = sin(x)
-    b = 2a
+    b = 0.2 + a
```

Comment:
Changed because, now we're not depending on ChainRules, we defined the rules here, and `+` has a simpler rule :)